README.md

```diff
@@ -1,8 +1,51 @@
-# Jarivs Data Engineering Training
-1. [Linux cluster monitoring agent (Linux and SQL)](./linux_sql) In-progress
-2. [Core Java Apps](./core_java) In-progress
-3. [Springboot Trading REST API](./springboot) In-progress
-4. [Hadoop/Hive](./hadoop) In-progress
-5. [Spark/Scala](./spark) In-progress
-6. [Cloud/DevOps](./cloud_devops) In-progress
+# Jaini Shah . Jarvis Consulting
 
+A passionate learner, detail-oriented, responsible and committed engineer graduated with a Masters of Applied Computing degree from University of Windsor. Having experience and understanding of agile scrum methodology, willing to work beyond the comfort zone to achieve milestones and aim to apply the skills in dynamic business environment to find innovative solutions for complex problems.
```

Comment:
Too short..Please see my inline comment below.


I'm a passionate learner, detail-oriented, responsible and committed engineer who graduated with a Masters of Applied Computing degree from the University of Windsor. I have experience and understanding of agile scrum methodology, willing to work beyond my comfort zone to achieve milestones and apply the skills in a dynamic business environment to find innovative solutions for complex problems. Please talk about your technical skill sets and what interest you in software development